scripts/dataplane_package/template/setup.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
```

Comment:
@msyyc can we take a step back and do this autorest issue first? https://github.com/Azure/azure-sdk-for-python/issues/21074#issuecomment-935062698

Sorry if this wasn't clear!

It's the one @lmazuel said is step 1. Sorry, should've made this more clear earlier.

As an aside, this part of the code looks copy-pasted from another script that does packaging. I'm not sure how @lmazuel wants this quickstart to look like in the end, but i think either way it would be better to call the other script from this script, instead of duplicating code

Reply:
Hi @iscai-msft, I am really working on the https://github.com/Azure/autorest.python/issues/1053. But after thinking, I think it is better to develop it with script first since we needn't rely on the new release of `autorest.python` for new function, which is more flexible. Once it is stable enough, we could move some function(like I [commented](https://github.com/Azure/autorest.python/issues/1053#issuecomment-964862928)) to `autorest.python` and simplify the quick-start doc.
As for the packaging, do you mean https://github.com/Azure/azure-sdk-for-python/tree/main/tools/azure-sdk-tools/packaging_tools/templates? It is for mgmt-plane as well. Maybe we could reuse it but I don't want to do that since there are many little difference. If we want to use other script, we need to change other script which is dangerous and high cost. I want to make the script more independent and stable.
@lirenhe tried with the tool and it could help to create basic package framework very quickly. Of course, we will continually optimize the quick start doc.